assets/javascripts/discourse/components/share-report.js.es6

```diff
@@ -0,0 +1,88 @@
+import {
+  default as computed,
+  on
+} from "ember-addons/ember-computed-decorators";
+
+export default Ember.Component.extend({
+  classNames: ["share-report"],
+
+  group: null,
+  query: null,
+  visible: false,
+
+  @computed("group", "query")
+  link() {
+    return (
+      window.location.protocol +
```

Comment:
Can we use `Discourse.BaseUrl` here? It might also look nicer using template strings instead of concatenation.